@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fuselage/fn_fuselageDrag.sqf

```diff
@@ -0,0 +1,69 @@
+params ["_heli", "_deltaTime", "_altitude", "_temperature", "_rho"];
+
+systemChat format ["Custom Drag!"];
+
+//MOVE TO CONFIG!
+private _aerodynamicCenter  = [0.0, -1.94, -1.73]; //m
+
+//MOVE TO CONFIG!
+private _fuselageAreaFront  =  6.254;
+private _fuselageAreaSide   = 22.332;
+private _fuselageAreaBottom = 33.129;
+
+private _fuselageDragCoefX  = 1.5;
+
+
+//MOVE TO CONFIG!                   PA   -40     0    40
+private _dragCoefTableY       = [[   0, 0.54, 0.60, 0.60],   //ft 
+                                 [2000, 0.80, 0.70, 0.55],   //ft
+                                 [4000, 1.02, 0.92, 0.85],   //ft
+                                 [6000, 1.20, 1.00, 1.06],   //ft
+                                 [8000, 1.65, 1.25, 1.70]];  //ft
+private _interpDragCoefTableY = [_dragCoefTableY, _altitude] call fza_fnc_linearInterp;
+_dragCoefTableY               = [[-40, _interpDragCoefTableY # 1],
+                                 [  0, _interpDragCoefTableY # 2],
+                                 [ 40, _interpDragCoefTableY # 3]];
+_interpDragCoefTableY         = [_dragCoefTableY, _temperature] call fza_fnc_linearInterp;
+private _fuselageDragCoefY    = _interpDragCoefTableY # 1;
+
+private _fuselageDragCoefZ  = 0.5;
+
+
+private _vecX = [1.0, 0.0, 0.0];
+private _vecY = [0.0, 1.0, 0.0];
+private _vecZ = [0.0, 0.0, 1.0];
+
+[velocityModelSpace _heli # 0, velocityModelSpace _heli # 1, velocityModelSpace _heli # 2] 
+    params ["_locVelX", "_locVelY", "_locVelZ"];
+
+private _dragX = -_fuselageDragCoefX * 0.5 * _rho * _fuselageAreaSide   * (_locVelX * _locVelX);
```

Comment:
While it is an equivalent approach, could it be simpler to approach this using a singular drag force vector rather than 3 - one for each dimension? We should be able to get some 'SIMD' performance improvements using vector functions in SQF more.
```sqf
private _drag =
    [ _fuselageDragCoefX * _fuselageAreaSide * (_locVelX * _locVelX)
    , _fuselageDragCoefY * _fuselageAreaFront * (_locVelY * _locVelY)
    , _fuselageDragCoefZ * _fuselageAreaBottom * (_locVelX * _locVelZ)
    ] vectorMultiply (-0.5 * _rho * _deltaTime);

_heli addForce [_heli vectorModelToWorld _drag, _aerodynamicCenter];
```